Doc/whatsnew/3.9.rst

```diff
@@ -739,6 +739,9 @@ Removed
   the ``__annotations__`` attribute instead.
   (Contributed by Serhiy Storchaka in :issue:`40182`.)
 
+* The :meth:`symtable.SymbolTable.has_exec` method has been removed. It was
+  deprecated since 2006. (Contributed by Batuhan Taskaya in :issue:`40208`)
```

Comment:
I suggest to document the old behavior: mention that the deprecated method always returned False.

Reply:
Added.